src/screens/Ticketing/Purchase/Overview/index.tsx

```diff
@@ -194,7 +197,11 @@ const PurchaseOverview: React.FC<OverviewProps> = ({
           <MessageBox
             isMarkdown={true}
             containerStyle={styles.warning}
-            message={t(PurchaseOverviewTexts.notInspectableTokenDeviceWarning)}
+            message={t(
+              PurchaseOverviewTexts.notInspectableTokenDeviceWarning(
+                getDeviceNameForCurrentToken(token, remoteTokens),
```

Comment:
What happens here if we cannot find a device name? Haven't there been cases where the device name is unknown?